pkg/api/types.go

```diff
@@ -3693,10 +3693,14 @@ const (
 	ResourceRequestsMemory ResourceName = "requests.memory"
 	// Storage request, in bytes
 	ResourceRequestsStorage ResourceName = "requests.storage"
+	// Local Storage request for scratch space, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
+	ResourceRequestsStorageScratch ResourceName = "requests.storage.kubernetes.io/scratch"
```

Comment:
I find both complicated. Can we just resort with `requests.scratch-storage` or something similar?